packages/crypto/src/primitives/ed25519.ts

```diff
@@ -477,28 +516,25 @@ export class Ed25519 {
    * not assess whether the key corresponds to a known entity or its security status (e.g., whether
    * it has been compromised).
    *
-   * Example usage:
-   *
+   * @example
    * ```ts
    * const publicKey = new Uint8Array([...]); // A public key in byte format
-   * const isValid = await Secp256k1.validatePublicKey({ key: publicKey });
+   * const isValid = Ed25519.validatePublicKey({ publicKey });
    * console.log(isValid); // true if the key is valid on the Edwards curve, false otherwise
    * ```
    *
-   * @param options - The options for the public key validation.
-   * @param options.key - The public key to validate, represented as a Uint8Array.
+   * @param params - The parameters for the public key validation.
+   * @param params.publicKey - The public key to validate, represented as a Uint8Array.
    *
    * @returns A Promise that resolves to a boolean indicating whether the key
    *          corresponds to a valid point on the Edwards curve.
    */
-  private static async validatePublicKey(options: {
-    key: Uint8Array
-  }): Promise<boolean> {
-    const { key } = options;
-
+  private static validatePublicKey({ publicKey }: {
```

Comment:
is this supposed to be public?

Reply:
Good question.  It was initially but was marked private when working on the new `Signer`, `AsymmetricKeyGenerator`, etc. types and attempting to get these classes to "satisfy" those type definition constraints.  While it technically worked, it had the unfortunate side effect of making TSDocs disappear (longer discussion about anonymous classes).  Will mark as public for now and see how much usage these functions get.